js/app.js

```diff
@@ -0,0 +1,178 @@
+//Easy Going
+
+// for (i = 1; i <= 20; i++) {
+// 	console.log(i)
+// }
```

Comment:
please do not submit homework with all the answers commented out :(

Reply:
commenting out while you're working makes sense, but uncomment before you submit plz thx

Reply:
also use `let` in your for loop